src/linear_algebra/basic.lean

```diff
@@ -800,9 +801,15 @@ span_le.2 $ subset.trans h subset_span
 lemma span_eq_of_le (h₁ : s ⊆ p) (h₂ : p ≤ span R s) : span R s = p :=
 le_antisymm (span_le.2 h₁) h₂
 
-@[simp] lemma span_eq : span R (p : set M) = p :=
+lemma span_eq : span R (p : set M) = p :=
```

Comment:
I'm quite surprised to see `simp` being removed here. Is there an easy explanation of why we shouldn't keep it?

Reply:
Yes, the linter complains! This is now proved via the new lemma